OptimizelySDK.Tests/ProjectConfigTest.cs

```diff
@@ -130,21 +138,44 @@ public void TestInit()
                     { "group_exp_1_var_2", Config.GetVariationFromKey("group_experiment_1", "group_exp_1_var_2") }
                  }
                 },
-                {
-                    "group_experiment_2", new Dictionary<string, object>
+                { "group_experiment_2", new Dictionary<string, object>
                  {
                      {"group_exp_2_var_1", Config.GetVariationFromKey("group_experiment_2", "group_exp_2_var_1") },
                      { "group_exp_2_var_2", Config.GetVariationFromKey("group_experiment_2", "group_exp_2_var_2") }
                  }
+                },
+                { "test_experiment_multivariate", new Dictionary<string, object>
+                 {
+                     {"Fred", Config.GetVariationFromKey("test_experiment_multivariate", "Fred") },
+                     { "Feorge", Config.GetVariationFromKey("test_experiment_multivariate", "Feorge") },
+                     { "Gred", Config.GetVariationFromKey("test_experiment_multivariate", "Gred") },
+                     { "George", Config.GetVariationFromKey("test_experiment_multivariate", "George") }
+                 }
+                },
+                { "test_experiment_with_feature_rollout", new Dictionary<string, object>
+                 {
+                     {"control", Config.GetVariationFromKey("test_experiment_with_feature_rollout", "control") },
+                     { "variation", Config.GetVariationFromKey("test_experiment_with_feature_rollout", "variation") }
+                 }
+                },
+                { "test_experiment_double_feature", new Dictionary<string, object>
+                 {
+                     {"control", Config.GetVariationFromKey("test_experiment_double_feature", "control") },
+                     { "variation", Config.GetVariationFromKey("test_experiment_double_feature", "variation") }
+                 }
+                },
+                { "test_experiment_integer_feature", new Dictionary<string, object>
+                 {
+                     {"control", Config.GetVariationFromKey("test_experiment_integer_feature", "control") },
+                     { "variation", Config.GetVariationFromKey("test_experiment_integer_feature", "variation") }
+                 }
                 }
             };
 
-
-
-            Assert.IsTrue(TestData.CompareObjects(variationKeyMap, Config.VariationKeyMap));
+            Assert.IsTrue(TestData.CompareObjects(expectedVariationKeyMap, Config.VariationKeyMap));
 
             // Check Variation ID Map
-            var variationIdMap = new Dictionary<string, object>
+            var expectedVariationIdMap = new Dictionary<string, object>
```

Comment:
Can you generate some variable usages within variations of one of the experiment and assert that the `FeatureVariableUsageInstance` maps are generated properly for those variations.